CMakeLists.txt

```diff
@@ -139,18 +173,14 @@ if (MSVC)
 endif()
 
 target_link_libraries(monitor
-  OpenDDS::Dcps
-  OpenDDS::Rtps
-  OpenDDS::Rtps_Udp
-  OpenDDS::Tcp
+  manager
```

Comment:
Having "manager" as a required dependency of "monitor" makes it so that an existing OpenDDS build may not be able to be used to build opendds-monitor (based on the C++ standard used for ACE+OpenDDS).

Reply:
I was just trying to avoid rebuilds of the manager code for the test apps. Is there a better way?

Reply:
I think there is, but I'm not familiar enough with how "manager" is used -- does anything in "monitor" include those header files?  I can look into it later.